minfraud/help.md

```diff
@@ -161,15 +166,19 @@ This action can be used to retrieve intelligence information for all query items
 |shipping_phone_country_code|string|None|False|Country code for phone number|None|
 |order_currency|string|None|False|Currency code for the currency used|None|
 |billing_country|string|None|False|Two character country code|None|
+<<<<<<< HEAD
+|payment_was_authorized|boolean|None|False|Payment authorized: true/false|None|
+=======
 |payment_was_authorized|boolean|None|False|Payment authorized\: true/false|None|
+>>>>>>> master
```

Comment:
remove all artifacts from git merge in minfraud help file

Reply:
That face looks more disappointed to me, than confused.  This is sloppy on my part, and I apologize.